pkg/scheduler/scheduler.go

```diff
@@ -536,6 +542,14 @@ func (sched *Scheduler) scheduleOne() {
 		return
 	}
 
+	// Run "reserve" plugins.
```

Comment:
I'm curious that are we going to replace current "assuming" logic as a `ReservePlugin`, and provided by default? Or, keep current logic as is.

Reply:
The assumption logic which updates the scheduler's cache remains as a part of core.